client/src/components/RelatedItems/Outfits.jsx

```diff
@@ -0,0 +1,175 @@
+import { useState, useEffect } from 'react';
+import Rating from '@mui/material/Rating';
+import { Virtual, Navigation, Pagination } from 'swiper/modules';
+import ReviewStars from '../common/ReviewStars';
+import { Swiper, SwiperSlide } from 'swiper/react';
```

Comment:
See issue #57, you will want to rewrite this natively without the help of an external package.